### PR TITLE
docs: fix typo in demo

### PR DESCRIPTION
**Pull request by xishanxu**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/tabs/demo/extra.md](https://github.com/xishanxu/ant-design/blob/master/components/tabs/demo/extra.md)

### CODE REVIEW

**Comment by xishanxu** [comment]
docs: tabs组件 附加内容的demo，其中position 及 direction单词错误修复 (Update extra.md )
fixed #26388

File: components/tabs/demo/extra.md [hunk 1]
```diff
@@ -30,16 +30,16 @@ const OperationsSlot = {
...
+    if (position.length === 0) return null;
 
-    return positon.reduce(
-      (acc, direaction) => ({ ...acc, [direaction]: OperationsSlot[direaction] }),
+    return position.reduce(
+      (acc, direction) => ({ ...acc, [direction]: OperationsSlot[direction] }),
       {},
     );
-  }, [positon]);
+  }, [position]);
```
Comment (afc163) [inline components/tabs/demo/extra.md:42]: 下面还有 positon

Comment (xishanxu) [reply components/tabs/demo/extra.md:42]: 稍等

**Comment by xishanxu** [comment]
已修改

**Review by xrkffgg — APPROVED** [review]

**Review by MrHeer — APPROVED** [review]
LGTM

**Pull request merged**